app/jinja_filters.py

```diff
@@ -311,6 +310,15 @@ def format_date_range_no_repeated_month_year(context, start_date, end_date, date
     return mark_safe(context, output)
 
 
+@evalcontextfunction
+def one_of(context, *args):
+    one_of_exists = [x for x in args if x]
```

Comment:
Does this guarantee order?
What if both are supplied?

Reply:
As long as the data structure preserves order. In our case yes, it should guarantee that it is the first valid element.